stdlib/influxdata/influxdb/wide_to.go

```diff
@@ -306,9 +307,17 @@ func (t *WideToTransformation) writeTable(tbl flux.Table) error {
 					continue
 				}
 
-				metric.Fields = append(metric.Fields, &lp.Field{
+				v := values.Unwrap(fieldVal)
+				if fv, ok := v.(float64); ok {
+					if math.IsNaN(fv) || math.IsInf(fv, 0) {
```

Comment:
Why do the check here at all? If you have to do the check in the dependency to provide a good diagnostic, and both `to` and `wide_to` will use that path, isn't that enough?

I see in your header comment that it should prevent the need to check in the dependency provider, but why is that bad?

Reply:
The checks don't provide error messages or diagnostics. They're just to skip the point.

The reason for updating the dependency is so that we don't need to add similar checks to every dependency implementation. I suspect that influxdb has its own local implementation and I don't want to go check if that one also causes an issue. The http writer is the path for most to writes, but not all.

We could feasibly remove this check from the dependency, but I just want to protect against potential problems in other sections of code that might use the dependency. I don't want to see a situation where they get an error message saying, "You can't write NaN or Inf" if we make an error in the `to()` function because then we're giving them bad information rather than learning we have an error in our own code. So I thought it was better to just do the check twice to ensure that we both protect bad dependency code and bad `to()` implementation code.